src/libraries/Common/tests/System/Net/Capability.Quic.cs

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Net.Quic;
+using System.Threading;
+
+namespace System.Net.Test.Common
+{
+    public static partial class Capability
```

Comment:
This can use a comment that says it is a workaround for bug in Lttng - see https://github.com/dotnet/runtime/issues/74795 for details, and explain how this workaround works.

It is not obvious to me how this guarantees that the Quic is initialized before any tests run. Would it be better to `[ModuleInitializer]` for this workaround?

Reply:
Static properties of the calls will be initialized when the `Capability` class is loaded, right? I can make direct reference to it (with comment) from the tests. But I did not want to touch every class. If we use `[ModuleInitializer]` we could do it for the test assembly right? 
I think this is one reason what "normal" HTTP test work - They have Unit conditionals that when be execute when XUnit prepares list of eligible tests. (and there is less `RemoteExecutor code`)

Reply:
> If we use [ModuleInitializer] we could do it for the test assembly right?

Right, `[ModuleInitializer]` runs before any other code in the assembly runs.

Reply:
updated. Running tests again over night.